tests/test_causal_model.py

```diff
@@ -1,5 +1,8 @@
 import pytest
 
+import os, sys
+sys.path.append(os.path.abspath("../"))
```

Comment:
remove `sys.path` line. That is only for local testing. Other people may have different local configuration.